src/Listener/PayPalOrderCompletedListener.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\PayPalPlugin\Listener;
+
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\PayPalPlugin\Processor\PayPalOrderCompleteProcessor;
+use Symfony\Component\Workflow\Event\CompletedEvent;
+use Webmozart\Assert\Assert;
+
+final class PayPalOrderCompletedListener
```

Comment:
```suggestion
final class CompletePayPalOrderListener
```
WDYT?

Reply:
`CompletePayPalOrderListener` is better name